common4j/src/main/com/microsoft/identity/common/java/cache/MsalCppOAuth2TokenCache.java

```diff
@@ -148,7 +148,7 @@ public synchronized void saveCredentials(@NonNull final Credential... credential
             }
         }
 
-        saveCredentialsInternal(credentials);
+        saveCredentialsInternal(true, credentials);
```

Comment:
>saveCredentialsInternal(true [](http://example.com/codeflow?start=8&length=28)

could we bubble this true/false decision up to this method's parameters? This seems risky enough that we'll want to guard it with a flight (even if it starts default-enabled), and I think we'll need that so our layer that understands the flights can make that choice.